docs/docs/guides/web3_config/index.md

```diff
@@ -201,44 +202,394 @@ console.log(web3.getContextObject().config)
 */
 ```
 
+## Explanation of Configuration Parameters
+
+### [handleRevert](/api/web3-core/class/Web3Config#handleRevert)
+The following methods will retrieve specific error types and error messages when `handleRevert` is set to `true`:
+- [`web3.eth.sendTransaction()`](/api/web3-eth/function/sendTransaction);
+- [`web3.eth.call()`](/api/web3-eth/function/call);
+- [`myContract.methods.myMethod().call()`](/libdocs/Contract#call);
+- [`myContract.methods.myMethod().send()`](/libdocs/Contract#send);
+
+The error types will be one of the following:
+- [InvalidResponseError](/api/web3-errors/class/InvalidResponseError)
+- [ContractExecutionError](/api/web3-errors/class/ContractExecutionError)
+- [TransactionRevertWithCustomError](/api/web3-errors/class/TransactionRevertWithCustomError)
+- [TransactionRevertedWithoutReasonError](/api/web3-errors/class/TransactionRevertedWithoutReasonError)
+- [TransactionRevertInstructionError](/api/web3-errors/class/TransactionRevertInstructionError)
+- [TransactionPollingTimeoutError](/api/web3-errors/class/TransactionPollingTimeoutError)
```

Comment:
from where you got these error types , that these could be thrown as result of revert checking?

Reply:
These errors are from the following code
https://github.com/web3/web3.js/blob/cbbbd84bac7051b57ec636b857e06adcc3da05f6/packages/web3-eth/src/utils/send_tx_helper.ts#L250-L256